packages/lesswrong/components/posts/PostsItemIcons.tsx

```diff
@@ -8,8 +8,8 @@ import { forumTypeSetting, isEAForum } from '../../lib/instanceSettings';
 
 const styles = (theme: ThemeType): JssStyles => ({
   iconSet: {
-    marginLeft: theme.spacing.unit,
-    marginRight: theme.spacing.unit,
+    marginLeft: isEAForum ? undefined : theme.spacing.unit,
+    marginRight: isEAForum ? 2 : theme.spacing.unit,
     lineHeight: "1.0rem",
     '&:empty': {
```

Comment:
The linkpost icon feels a bit too prominent now, a lot of the posts are linkposts so it isn't conveying that much information

![Screenshot 2023-03-10 at 15 03 18](https://user-images.githubusercontent.com/77623106/224349959-ccdab9a9-72f7-4c7f-bd44-2f0fdc8999fd.png)

Reply:
I agree, I'd actually be in favour of removing them fully and see if people miss them cc @oetherington

Reply:
Done!